synapseclient/services/json_schema.py

```diff
@@ -144,9 +140,10 @@ def expand(self):
     def bind_to_object(self, synapse_id: str):
         """Bind schema to an entity
 
-        :param synapse_id: Synapse Id to bind json schema to.
-        :type synapse_id:  str
+        Args:
```

Comment:
I have usually preferred to use `Arguments:` over `Args:`, but they both work as far as mkdocs is concerned.

Reply:
Yea, they both work but I will convert to use `Arguments`. `Args` is automatically generated by autoDocstring.